modules/ROOT/pages/build-app-practices.adoc

```diff
@@ -0,0 +1,115 @@
+= Best Practices to Build Mule Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+As you build your applications, consider the following best practices for creating a clean, organized, and sustainable project. These practices include guidelines on using Anypoint Studio, avoiding certain building app issues, and specific settings for using Mule runtime engine (Mule) components and connectors.
+
+== Anypoint Studio Best Practices
+If you use Anypoint Studio to build Mule apps, consider the following practices:
+
+* Update to the latest version of Anypoint Studio.
+* Use the recommended Java version (OpenJDK) for the specific Studio version.
+* Include the Mule Maven plugin for automated deployment (executed by CI/CD processes).
+
+=== Define a Project Structure
+To help maintain a clean, structured, and readable project in Anypoint Studio, consider the following recommendations:
+
+* Create a separate XML for global elements, for example configuration elements.
+* Create a separate XML for each use case or resource implementation.
+* Create separate XML files for common structures or logic.
+* Create different packages for resources such as DataWeave, WSDLs, examples, and so on.
+
```

Comment:
Does your audience know how the fourth list item, different packages, relates to the three previous items about separate XML or XML files?